docs/src/testbeds.md

```diff
@@ -0,0 +1,88 @@
+# Testbeds
+
+Stylix provides a suite of virtual machines which can be used to test and
+preview themes without installing the target to your live system.
+
+These can be particularly helpful for:
+
+- Working on targets before the login screen, since you can avoid closing
+  your editor to see the result.
+- Developing for a different desktop environment than the one you normally use.
+- Reducing the risk of breaking your system while reviewing pull requests.
+
+Testbeds are also built by GitHub Actions for every pull request. This is less
+beneficial compared to running them yourself, since it cannot visually check
+the theme, however it can catch build failures which may have been missed
+otherwise.
+
+## Creation
+
+New testbeds are defined by creating a file called `testbed.nix` within the
+folder for the corresponding target. This file will automatically be loaded
+as a NixOS module, with options such as `stylix.image` already defined.
+The module should include any options necessary to install the target and
+any supporting software - for example, a window manager.
+
+If the target can only be used through Home Manager, you can write a
+Home Manager module within the NixOS module using the following format:
+
+```nix
+{
+    home-manager.sharedModules = [{
+        # Write Home Manager options here
+    }];
+}
+```
+
+Using `home-manager.sharedModules` is preferred over `home-manager.users.guest`
+since it allows us to easily change the username or add additional users in
+the future.
+
+Once the module is complete, use `git add` to track the file, then the new
+packages will be available to use with `nix run` as described below.
```

Comment:
In personal experience I've often seen people run into things not working because they didn't run `git add`, so Nix didn't see the new file. This is even more confusing when we have automatic imports, since a file which wasn't added will simply not be imported rather than causing an error that the import doesn't exist.

As it's only one sentence I think it's worth keeping in.

Reply:
How about simply `[available to use](#usage)`? Since the usage section describes the exact command.

Reply:
> How about simply `[available to use](#usage)`? Since the usage section describes the exact command.

Sounds good. We could also do the following. Although, I don't think those differences are that important.

```suggestion

Once the module is complete, use `git add` to track the file, and then the new
packages will be [available](#usage).
```

Reply:
OK, I went for `[available to use](#usage)`.